MCDataProducts/inc/KalSeedMC.hh

```diff
@@ -34,15 +34,17 @@ namespace mu2e {
     MCRelationship _rel; // relationship of this particle to its primary
     uint16_t _nhits; // number of associated StrawHits
     uint16_t _nactive; // number of associated active hits
-    XYZVectorF _mom; // initial momentum
+    XYZTVectorF   _mom; // initial momentum
+    XYZVectorF _pos; // initial position
```

Comment:
To make it clear the _time value goes with the _pos value you should use
XYZTvectorF to store them as a 4-vector.